cypress/fixtures/admin_portal/coupon_creation_data.json

```diff
@@ -37,7 +37,7 @@
         "code": "",
         "id": null,
         "price": "0",
-        "quantity": "1",
+        "quantity": "3",
```

Comment:
you shouldn't be changing this data, instead update the value after fetching a single coupon data in a dict